packages/webpack-config-anansi/src/base/index.js

```diff
@@ -32,6 +36,10 @@ export default function makeBaseConfig({
       cacheCompression: mode === 'production',
       compact: mode === 'production',
       ...babelLoaderOptions,
+      caller: {
+        hasJsxRuntime: semver.gte(react.version, '16.14.0'),
```

Comment:
also react 15.7 and 0.14.10

Reply:
I don't think I really support react <16. There's probably other stuff that breaks too